sdk/src/RealUserMonitor.ts

```diff
@@ -150,11 +163,12 @@ export default class RealUserMonitor {
    * This method sends a mobile event timing message to the raygun server. If the current session
    * has not been setup, this method will also ensure that the session has been allocated an ID
    * before sending away any data.
-   * @param name - Name of the event (specific to the event).
-   * @param duration - How long the event took.
+   * @param payload
    */
-  async sendViewLoadedEvent(name: string, duration: number) {
-    if (!this.curRUMSessionId) {
+  async sendViewLoadedEvent(payload: Record<string, any>) {
+    const {name, duration} = payload;
```

Comment:
Nit: This assignment could be performed after the current sessionID check, because they aren't use if that condition evaluates to true